src/components/lava-coder/vent-location-popup.tsx

```diff
@@ -0,0 +1,98 @@
+import { Cartesian3, Cartographic, CesiumWidget, Math as CSMath } from "@cesium/engine";
+import React, { useEffect, useRef, useState } from "react";
+import CopiedButtonIcon from "../../assets/lava-coder/content-copied-icon.png";
+import CopyButtonIcon from "../../assets/lava-coder/content-copy-icon.png";
+import IconButton from "../buttons/icon-button";
+
+import "./vent-location-popup.scss";
+
+interface IProps {
+  viewer: CesiumWidget | null;
+  ventLocation: Cartographic | null;
+  verticalExaggeration: number;
+  show: boolean;
```

Comment:
`show` seems a bit confusing to me. `visible`? `hidden`?

Reply:
It's fairly standard for popups and dialogs, so I think I'll leave it for now.